### PR TITLE
implement management mockup

### PR DESCRIPTION
**Pull request by pldespaigne**
download

### CODE REVIEW

File: apps/movie-financing/movie-financing/src/app/financing/card-blacklist/card-blacklist.component.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  styleUrls: ['./card-blacklist.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class CardBlacklistComponent implements OnInit {
+  @Input() investor: Investor;
+
+  constructor() {
+  }
+
+  ngOnInit() {
```
Comment (phpgeekfr) [inline apps/movie-financing/movie-financing/src/app/financing/card-blacklist/card-blacklist.component.ts]: Empty `ngOnInit`

File: apps/movie-financing/movie-financing/src/app/financing/card-investor/card-investor.component.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  styleUrls: ['./card-investor.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class CardInvestorComponent implements OnInit {
+  @Input() investor: Investor;
+
+  constructor() {
+  }
+
+  ngOnInit() {
```
Comment (phpgeekfr) [inline apps/movie-financing/movie-financing/src/app/financing/card-investor/card-investor.component.ts]: Empty `ngOnInit`

File: apps/movie-financing/movie-financing/src/app/financing/card-blacklist/card-blacklist.component.ts (hunk 1 above)
Comment (pldespaigne) [reply apps/movie-financing/movie-financing/src/app/financing/card-blacklist/card-blacklist.component.ts]: ✔️

File: apps/movie-financing/movie-financing/src/app/financing/card-investor/card-investor.component.ts (hunk 2 above)
Comment (pldespaigne) [reply apps/movie-financing/movie-financing/src/app/financing/card-investor/card-investor.component.ts]: ✔️

**Pull request merged**